README.md

```diff
@@ -0,0 +1,325 @@
+----
+
+## Overview
+
+OpenWrt is a highly extensible GNU/Linux distribution for embedded devices. Instead of trying to create a single, static firmware,
+OpenWrt provides a fully writable filesystem with optional package management. This frees you from the restrictions of the application
+ selection and configuration provided by the vendor and allows you to use packages to customize an embedded device to suit any application.
+For developers, OpenWrt provides a framework to build an application without having to create a complete firmware image and distribution around it.
+For users, this means the freedom of full customisation, allowing the use of an embedded device in ways the vendor never envisioned.
+
+Creator is MIPS based platform series from Imagination and Marduk is the latest board under that series. Pistachio is the name of Imagination's SoC based on MIPS.
+This guide helps as a quick start but for full details please see the OpenWrt projects
+own [wiki]("http://wiki.openwrt.org/doc/start").
+
+## Revision History
+
+| revision  | changes from previous revision |
+|---------- |------------------------------- |
+| 1.0.0     | Initial release                |
+
+
+
+## Package Content
+
+The release distribution is structured as documented in the upstream project however
+there are some key paths where IMG have added solutions:
+
+| Folder              				| Content                                              							|
+| :----               				| :----                                                							|
+| target/linux/pistachio			| IMG pistachio SoC based board configs and makefiles   						|
+| target/linux/pistachio/config-4.1		| IMG pistachio SoC specific kernel config							        |
+| target/linux/pistachio/creator-platform.config| IMG creator marduk platform specific openwrt config							| target/linux/pistachio/marduk/profiles/maruk.mk	| IMG pistachio SoC based Marduk platform profile			|
+
+## Getting started
+
+Firstly, to obtain a copy of the (Ci40) Marduk platform supported OpenWrt source code:
+
+ * Sign up for a Github account
+
+ * Install Git:  ```` sudo apt-get install git ````
+
+ * Clone the repository: ```` git clone https://github.com/FlowM2M/openwrt.git ````
+
+To simply make a build based on the IMG default config run the following commands:
+
+    $ cd openwrt
+    $ ./scripts/feeds update -a
+    $ ./scripts/feeds install -a
+_Ignore any "WARNING: No feed for package..." from the install feeds step._
+
+Load Marduk platform specific openwrt configuration for Pistachio.
+
+    $ make menuconfig
+
+1. Select the "Target System" as IMG MIPS Pistachio
+
+    Target System (Atheros AR7xxx/AR9xxx) --->(X) IMG MIPS Pistachio
+
+2. Check the "Target Profile" is set to Basic platform profile for Marduk
+
+    Target Profile (Basic platform profile for Marduk)  ---> (X) Basic platform profile for Marduk
```

Comment:
Now two profiles have been added
Marduk with cc2520 and Marduk with ca8210. Need to reflect that.